tests/Tests.elm

```diff
@@ -975,4 +975,49 @@ all =
                     joinOn Tuple.pair identity identity [ 1, 3, 2 ] [ 2, 1, 3 ]
                         |> Expect.equal [ ( 3, 3 ), ( 2, 2 ), ( 1, 1 ) ]
             ]
+        , describe "stoppableFoldl"
+            [ fuzz (list int) "behaves like foldl if function always returns Continue" <|
+                \xs ->
+                    stoppableFoldl (\n acc -> Continue (n + acc)) 0 xs
+                        |> Expect.equal (List.foldl (\n acc -> n + acc) 0 xs)
+            , test "simple example" <|
+                \() ->
+                    stoppableFoldl
+                        (\n acc ->
+                            if acc >= 50 then
+                                Stop acc
+
+                            else
+                                Continue (n + acc)
+                        )
+                        0
+                        (List.range 1 1000)
+                        |> Expect.equal 55
+            ]
+        , describe "stoppableFoldr"
+            [ fuzz (list int) "behaves like stoppableFoldl with reversed list" <|
+                \xs ->
+                    stoppableFoldr
+                        (\n acc ->
```

Comment:
To make it clearer and/or less error-prone, you could put the function inside a let declaration, so that it's very clear that both operations do the same thing.